src/k8s-engine/kim.js

```diff
@@ -0,0 +1,173 @@
+'use strict';
+
+// This file should probably be an instance of ImageManager < EventEmitter.
+// We'll get there eventually
+
+const { EventEmitter } = require('events');
+const process = require('process');
+const { spawn } = require('child_process');
+const os = require('os');
+const fs = require('fs');
+const path = require('path');
+const util = require('util');
+const paths = require('xdg-app-paths')({ name: 'rancher-desktop' });
+const resources = require('../resources');
+const K8s = require('./k8s');
+
+const REFRESH_INTERVAL = 5 * 1000;
+
+async function runCommand(args) {
```

Comment:
I believe this function is equivalent to:
```javascript
async function runCommand(args) {
  return await util.promisify(child_process.execFile)(resources.executable('kim'), args);
}
```
Complete with returning an object with `stdout` and `stderr` properties, or throwing if the command exits with non-zero…

Unrelated:
This may be more ergonomic as:
```javascript
async function runCommand(...args) {
```
That way the caller doesn't need to pass in an array.

Reply:
This suggestion might be outdated with the `kim-process-output` events firing while the process run.